hack/deployer/runner/gke.go

```diff
@@ -268,14 +268,27 @@ func (d *GKEDriver) create() error {
 	}
 	labels = fmt.Sprintf("%s,%s", strings.Join(toList(elasticTags), ","), labels)
 
-	return exec.NewCommand(`gcloud beta container --quiet --project {{.GCloudProject}} clusters create {{.ClusterName}} ` +
+	createGKEClusterCommand := `gcloud beta container --quiet --project {{.GCloudProject}} clusters create {{.ClusterName}} ` +
 		`--labels "` + labels + `" --region {{.Region}} --no-enable-basic-auth --cluster-version {{.KubernetesVersion}} ` +
 		`--machine-type {{.MachineType}} --disk-type pd-ssd --disk-size 40 ` +
 		`--local-ssd-count {{.LocalSsdCount}} --scopes {{.GcpScopes}} --num-nodes {{.NodeCountPerZone}} ` +
 		`--addons HorizontalPodAutoscaling,HttpLoadBalancing ` +
 		`--no-enable-autoupgrade --no-enable-autorepair --enable-ip-alias --metadata disable-legacy-endpoints=true ` +
 		`--network projects/{{.GCloudProject}}/global/networks/default ` +
-		strings.Join(opts, " ")).
+		strings.Join(opts, " ")
+	createAutopilotClusterCommand := `gcloud beta container --quiet --project {{.GCloudProject}} clusters create-auto {{.ClusterName}} ` +
+		`--region {{.Region}} --cluster-version {{.KubernetesVersion}} ` +
```

Comment:
Shouldn't we keep the labels?
```suggestion
		`--labels "` + labels + `" --region {{.Region}} --cluster-version {{.KubernetesVersion}} ` +
```

Reply:
Yeah I wondered about this as well

https://cloud.google.com/kubernetes-engine/docs/how-to/creating-managing-labels#creating_a_cluster_with_labels

I don't see you can create an autopilot cluster with labels using the cli. Let me take another look at this. Maybe we can update after creation for ap.

Reply:
I've updated to update autopilot labels after creation and tested, verifying that everything works as intended.